.changeset/orange-books-train.md

```diff
@@ -0,0 +1,5 @@
+---
+'@farfetched/core': patch
+---
+
+feat(core): simplify connect query declaration
```

Comment:
I suggest putting something meaningful for changelogs here. Like "Fixed incorrect type inference in ..."